galley/pkg/config/schema/schema_test.go

```diff
@@ -320,7 +320,7 @@ func TestSchemaBasic(t *testing.T) {
 		collection.NewName("k8s/networking.istio.io/v1alpha3/virtualservices"),
 	}))
 
-	g.Expect(s.KubeSource().Resources()[0].CanonicalResourceName()).To(Equal("networking.istio.io.v1alpha3/VirtualService"))
+	g.Expect(s.KubeSource().Resources()[0].CanonicalResourceName()).To(Equal("networking.istio.io/v1alpha3/VirtualService"))
```

Comment:
is there any risk in changing the format?

Reply:
This is a friendly name for logging/debugging purposes. We can change it over time as needed.